doc/subsystems/networking/gptp_release_notes.rst

```diff
@@ -0,0 +1,55 @@
+gPTP stack for Zephyr
+#####################
+
+Overview
+********
+
+This gPTP stack supports the protocol and procedures as defined in
+the IEEE 802.1AS-2011 standard (Timing and Syncronization for
```

Comment:
Add a link to the standard:

    the `IEEE 802.1AS-2011 standard`_ (Timing and Synchronization for
    Time-Sensitive Applications in Bridged Local Area Networks).

    .. IEEE 802.1AS-2011 standard: 
       https://standards.ieee.org/findstds/standard/802.1AS-2011.html